index.html

```diff
@@ -8,7 +8,7 @@
    <title>Wezesha</title>
  </head>
  <body>
-  <nav class="navbar">
+  <div class="navbar">
     <!-- LOGO -->
     <div class="logo">Wezesha</div>
```

Comment:
## Semantic tags
- semantic tags are a not-so-recent addition they are intended to make enhance the client's user experience. In this case the more appropriate code would be
-

Reply:
While you did a good job implementing all the required comments :+1: please ensure to always state explicit reasons why you requested a change; it might be obvious to you and not so to others.